### ISSUE TITLE
Updating Data

### ISSUE THREAD

**Issue by aerinvdz** [giero/LGoH-deck-suggester#12]
Cannot update data after entering to spreadsheet.  In the "All" worksheet there is an affinity "N/A" which the updater says is "Invalid value or number of stars"

The "All" worksheet is locked so can't edit to fix it.  :)

**Comment by giero** [comment]
Yeah, that's why it's dangerous when you allow to edit things like this spreadsheet for everyone.

I removed column with CODEX #, because it's useless for my app. It's broken also, because different heroes can have same id...).

Should be fine now :)

**Comment by aerinvdz** [comment]
Thanks!I feel bad for sending so many messages whenever I try to get it to update.Would you rather I didn’t go in and do entries?

I’ve probably bugged you more than anyone else since I started using it.:)

Melissa

Sent from XFINITY Connect Mobile App



------ Original Message ------

**Comment by giero** [comment]
Send as much messages as you want :) I'm not playing this game anymore but I can still help you all with my app with any issue.

And if you can, update this spreadsheet as much as you can, because it depends now on players to keep it up to date :)

**Comment by aerinvdz** [comment]
I’ll update what I can.I’ve only been playing a few months so most of my cards are the newer ones.I do have a couple that are missing stats in the spreadsheet though, so I’ll at least add those.

Thanks for being willing to keep it going.I’ve been pointing the newer guild members here to help them out.

Melissa

Sent from XFINITY Connect Mobile App



------ Original Message ------

**Comment by djm61** [comment]
EDIT: fixed the issue, had to go digging but someone entered invalid data

I updated the spreadsheet and I've run into an issue.  The error I get when I hit the "Update Now" button is: ... and it's failed for [dark] Escalatera, Precision Shield ****

Invalid counter skill (Dispel Alacrity IV (?))

I removed the entry for the extra Escalatera, but it's still failing to update with the same message.  Not sure what is going on.

**Comment by djm61** [comment]
I have found another issue, I'm adding the current event hero (Guru Pachycept) and he's a 7x commander at 5* and a 10x at 6*.  The spreadsheet wont let me enter more than 5x.  Can it be opened up to allow 7x and 10x?

**Comment by aerinvdz** [comment]
The issue is that the 7x and 10x are only for the event.Once the event is over the cards go back to their normal multipliers.

Melissa

Sent from XFINITY Connect Mobile App



------ Original Message ------

**Comment by djm61** [comment]
ahh, i was unaware of that, thanks!

**Comment by chudsrevenge** [comment]
Hi I don't know if you're still working on issues related to the deck builder, but I wanted to reach out to you anyways. Similar to issues above the spreadsheet is not updating in the actual deck builder after updating. It also gives errors when trying to update from the spreadsheet. Thanks.

**Comment by aerinvdz** [comment]
Specifically, Elf needs to be added as a race. The most recent Water event heroes were Elf.

Melissa

**Comment by giero** [comment]
Well - I don't play this game anymore, so I am not aware of any game mechanic changes, but I can still add new races or fix spreadsheet issues (if it's not completely messed up ;])

Elves are added - thanks for info about this.

**Comment by aerinvdz** [comment]
I was just clarifying what the other person’s problem was so you wouldn’t have to dig too deep.:)

They have significantly changed how decks work and how damage is done over the last few months.It’s quite possible the algorithms are totally different than previously.

I’m playing less and less as well due to some changes they’ve been making.

Melissa

Sent from XFINITY Connect App



------ Original Message ------
